### ISSUE TITLE
Insufficient scope for getUser()

### ISSUE THREAD

**Issue by datenfalke** [rhysnhall/etsy-php-sdk#24]
I have authorized my app and I can get access tokens. Refreshing is also working.

As soon as I do $etsy->getUser() I get an "insufficient_scope" error, and  I cannot do a $user->getShop() as written in the readme.

Using Postman with the bearer token and the x-api-key header parameter I get the same "insufficient_scope" error if I try https://api.etsy.com/v3/application/users/xxxxxxxxx with xxxxxxxx = user_id or https://api.etsy.com/v3/application/users/me .

I can although do https://api.etsy.com/v3/application/users/xxxxxxxxx/shops in Postman and see the shop_id.

I can than do a https://api.etsy.com/v3/application/shops/yyyyyyyy with yyyyyyyy = the shop_id what is basically 
what $user->getShop() is doing?

Is there a problem with my old API account which I already used for API v2?

In the end I want to create and update listings to sync my Etsy shop with my normal online shop.

**Comment by rhysnhall** [comment]
It sounds like you have not set the correct scopes. Your existing app registered for v2 should also work with v3.
Refer to their [documentation on scopes](https://developers.etsy.com/documentation/essentials/authentication#scopes), or even contact the Etsy team.

**Comment by datenfalke** [comment]
Sorry I forgot to mention that I a have the all scopes activated with 
`$scopes = \Etsy\Utils\PermissionScopes::ALL_SCOPES;`
I also get the complete scopes shown when authorizing the app.

**Comment by datenfalke** [comment]
If I check the scope of the access token I only get, as you said:
```
{
    "scopes": [
        "listings_d",
        "listings_r",
        "listings_w"
    ]
}
```

If I look here https://www.etsy.com/de/your/apps I have multiple authorizations also including some with only the listing scopes activated.

I revoked now all authorizations except one with all scopes.

In the moment only the 3 listing scopes still work. I tried also the DELETE Listing endpoint, which worked.

The GET User endpoint still does not work because of missing scope. I think the missing scope is email_r (Read your email address) as mentioned here https://developers.etsy.com/documentation/reference/#operation/getUser

I ask the Etsy support what I am missing.
